src/applications/vaos/new-appointment/components/VAFacilityPage/VAFacilityPageV2.jsx

```diff
@@ -68,19 +61,43 @@ export default function VAFacilityPageV2() {
     schema,
     selectedFacility,
     showEligibilityModal,
+    showVariant,
     singleValidVALocation,
     sortMethod,
     typeOfCare,
   } = useSelector(state => getFacilityPageV2Info(state), shallowEqual);
+  const sortOptions = [
```

Comment:
@jenniemc The `Select` documentation doesn't make it clear to me, but it also accepts options in this way. Allowed me to refactor that sort find method we wrote.